### PR TITLE
[Main Course] - Tensorflow Handbook

### PR DESCRIPTION
**Pull request by jhqian0909**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Post for Tensorflow Handbook

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Main_Course/Deep_Learning/2021-12-01-Tensorflow-Doc.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+张量可以理解为多维数组，是TensorFlow中的基本数据结构。  
+
+计算图就是整个计算关系。TensorFlow1.0采用的是静态计算图，在创建完计算图后，需要开启一个session才会显式执行。进入TensorFlow2.0后，为了方便调试，TensorFlow采用了动态计算图。由于动态计算图效率会低一些，TensorFlow允许我们使用@tf.function装饰器构建静态计算图，也被称作Autograph。  
+
+神经网络在更新权重时，很重要的一步是求解梯度。TensorFlow提供了tf.GradientTape(梯度磁带)，使我们可以很方便的求解梯度，更新网络。
+
+## 4 层次结构
+
+第三部分介绍了TensorFlow的层次结构，主要是低、中、高三阶的api示例。  
+低阶api直接操作张量、计算图和自动微分。虽然显得有些复杂和原始，但是是我们离开新手村，自定义模型时不可获取的知识点。
```
Comment (Realvincentyuan) [inline docs/Main_Course/Deep_Learning/2021-12-01-Tensorflow-Doc.md:35]: API需要大写

**Review by Tina0330 — APPROVED** [review]
感谢推荐，又多了一本学习资料! 谢谢佳辉 ：）

**Review by Realvincentyuan — APPROVED** [review]
LGTM

File: docs/Main_Course/Deep_Learning/2021-12-01-Tensorflow-Doc.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+template: overrides/blogs.html
+---
+
+# 读《30天吃掉那只TensorFlow2》
+
+!!! info
+    作者：Void，发布于2021-12-01，阅读时间：约6分钟，微信公众号文章链接：[:fontawesome-solid-link:](https://mp.weixin.qq.com/s/z2uBxwe8UNDXWMDNS_k-Gg)
```
Comment (Tina0330) [inline docs/Main_Course/Deep_Learning/2021-12-01-Tensorflow-Doc.md:8]: 到时候注意下公众号链接哦

**Review by EchoTang04 — APPROVED** [review]
LGTM, thank you Jiahui!

File: docs/Main_Course/Deep_Learning/2021-12-01-Tensorflow-Doc.md (hunk 1 above)
Comment (jhqian0909) [reply docs/Main_Course/Deep_Learning/2021-12-01-Tensorflow-Doc.md]: 已修改

[the remark this answers is not in the document]
Comment (jhqian0909) [reply docs/Main_Course/Deep_Learning/2021-12-01-Tensorflow-Doc.md]: 已修改

**Pull request merged**